js/adapt-offlineStorage-xapi.js

```diff
@@ -58,7 +58,7 @@ define([
     },
 
     /**
-     * @returns {{name: string in the format Firstname Lastname, firstname: string, lastname: string }}
+     * @returns {{id: string, name: string, firstname: string, lastname: string}} The learner's id (Actor.account.name), full name (in the format Firstname Lastname), first and last names
```

Comment:
comment needs updating following f1aa6d6

```suggestion
     * @returns {{id: string, name: string, firstname: string, lastname: string}} The learner's id, full name (in the format Firstname Lastname), first and last names
```